public/class-animalshelter-post.php

```diff
@@ -65,12 +65,12 @@ public function is_child(): bool {
 		return false;
 	}
 
-	public function get_publication_date( $dateFormat = 'U' ): bool|int|string {
-		return get_the_time( $dateFormat, $this->id );
+	public function get_publication_date( $dateFormat = 'U' ): int {
+		return (int)get_the_time( $dateFormat, $this->id );
```

Comment:
String

Reply:
Done!!! Can you review the issue again, please??

Reply:
Thanks! :heart: